File: src/main/java/fr/vlegall/sochief/repository/RecipeDifficultyRepository.java
package fr.vlegall.sochief.repository;

import fr.vlegall.sochief.model.recipe.RecipeDifficulty;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

public interface RecipeDifficultyRepository extends JpaRepository<RecipeDifficulty, Long> {
    List<RecipeDifficulty> findAllByOrderByIdAsc();
}

File: src/main/kotlin/fr/vlegall/sochief/repository/RecipeDifficultyRepository.kt
package fr.vlegall.sochief.repository

import fr.vlegall.sochief.model.recipe.RecipeDifficulty
import org.springframework.data.jpa.repository.JpaRepository

interface RecipeDifficultyRepository : JpaRepository<RecipeDifficulty, Long> {
    fun findAllByOrderByIdAsc(): List<RecipeDifficulty>
}
